_projects/model-my-watershed.md

```diff
@@ -0,0 +1,26 @@
+---
+# Feel free to add content and custom Front Matter to this file.
+# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
+
+layout: project
+
+# NO EDITING ABOVE THIS LINE
+# # # # # #
+
+seo_description: "GeoTrellis runs the hydrologic models and returns results to students as they make simulated changes to the landscape in the Model My Watershed application, created with the Stroud Water Resource Center."
```

Comment:
: "GeoTrellis runs hydrologic models and returns results to students as they mak changes to the simulated landscape in the Model My Watershed application, created with the Stroud Water Resource Center."

Reply:
Also fixed my run-on sentence.